api/src/main/java/org/hawkular/accounts/api/internal/CassandraSessionInitializer.java

```diff
@@ -67,12 +68,19 @@ public void init() {
         sessionFuture = executor.submit(cassandraSessionCallable);
     }
 
+    @PreDestroy
+    public void destroy() throws Exception {
+        logger.shuttingDownCassandraDriver();
+        sessionFuture.get().getCluster().closeAsync();
```

Comment:
Would it be better to use the blocking version, e.g.,  ```sessionFuture.get().getCluster().close()``` to avoid any possible race conditions?